music21/romanText/tsvEgOp18No1Mov3.tsv

```diff
@@ -0,0 +1,204 @@
+"chord"	"altchord"	"measure"	"beat"	"totbeat"	"timesig"	"op"	"no"	"mov"	"length"	"global_key"	"local_key"	"pedal"	"numeral"	"form"	"figbass"	"changes"	"relativeroot"	"phraseend"
```

Comment:
is this file copyright cleared to include?

Reply:
Modified from of the docs in this corpus / under this license:
https://github.com/DCMLab/ABC/blob/master/LICENSE.md

Attribution credit included in the code, but not on the file itself.

If that's a problem, I'll produce a new analysis in this format give my permission.

Reply:
Ah, if it's a non-commercial license then it'll need to be moved to the "corpus/demos" directory and the license information in the corpus (I can't remember where it is) needs to be updated to list it.  `music21` w/o corpus needs to be usable for all commercial purposes.

Reply:
New file it is!